### DISCUSSION TITLE
Jobs don't schedul

### DISCUSSION THREAD

**Discussion by Shukhratbek-Kholmatov** [agronholm/apscheduler#756] [Q&A]
I wrote aiogram bot and it opens group in time and closes in time,i used apscheduler. when I run script on server it works normally but after 1 day jobs don't schedul.Please help bro!

Below my zip code file.
[group_restricter_bot.zip](https://github.com/agronholm/apscheduler/files/12035363/group_restricter_bot.zip)

**Comment by agronholm** [comment]
What have you done to debug this? Did you follow the troubleshooting instructions in the documentation?

**Reply by Shukhratbek-Kholmatov** [reply]
i don't know what to do,everything has written from documentation

**Reply by agronholm** [reply]
I'm sorry, that sentence makes no sense. Have you read the troubleshooting section? If so, what does the debug log indicate?

**Reply by Shukhratbek-Kholmatov** [reply]
give me link bro

**Reply by agronholm** [reply]
https://apscheduler.readthedocs.io/en/3.x/userguide.html#troubleshooting
